gsa/src/web/pages/reports/alertactions.js

```diff
@@ -51,27 +57,32 @@ class AlertActions extends React.Component {
 
     this.state = {
       alerts: [],
+      showTriggerAlertDialog: false,
     };
 
     this.handleAlertChange = this.handleAlertChange.bind(this);
-    this.handleTestAlert = this.handleTestAlert.bind(this);
+    this.handleTriggerAlert = this.handleTriggerAlert.bind(this);
     this.onAlertCreated = this.onAlertCreated.bind(this);
+    this.handleOpenTriggerAlertDialog = this.handleOpenTriggerAlertDialog.bind(this); /* eslint-disable-line max-len */
+    this.handleCloseTriggerAlertDialog = this.handleCloseTriggerAlertDialog.bind(this); /* eslint-disable-line max-len */
+    this.handleValueChange = this.handleValueChange.bind(this);
   }
 
   componentDidMount() {
     const {gmp} = this.props;
+    this.props.loadReportComposerDefaults();
     gmp.alerts.getAll().then(response => {
```

Comment:
Because the alerts are only necessary when opening the composer now, maybe be should load them only on demand when opening the composer dialog. Also it would be better to load them from the store.